### PR TITLE
Add card-text for latest devices supported

### PR DESCRIPTION
**Pull request by mendel5**
While the section "Devices supported by the LVFS" on https://fwupd.org/lvfs/devices/ has a card-text, the section "Latest Devices Supported" does not. To increase consistency I would like to add a card-text for this section as well.

### CODE REVIEW

File: lvfs/devices/templates/devicelist.html [hunk 1]
```diff
@@ -22,6 +22,10 @@ <h2 class="card-title">Search for devices</h2>
 <div class="card mt-3">
   <div class="card-body">
     <h2 class="card-title">Latest Devices Supported</h2>
+    <p class="card-text">
+      This list shows the 6 latest devices supported by the LVFS.
+      This list is automatically generated and will be updated when new devices are supported.
```
Comment (hughsie) [inline lvfs/devices/templates/devicelist.html:27]: Like the concept, but the `supported` word is repeated and it seems a bit wordy. Perhaps `This list shows the 6 more recent devices supported by the LVFS and is automatically updated.`

Comment (mendel5) [reply lvfs/devices/templates/devicelist.html:27]: Thinking about this again, it seems that it's kind of obvious that the list is updated automatically and not manually. So maybe we don't need that sentence at all. How about just this:

`This list shows the 6 latest devices supported by the LVFS.`

Comment (hughsie) [reply lvfs/devices/templates/devicelist.html:27]: > it seems that it's kind of obvious that the list is updated automatically

Agree!

> This list shows the 6 latest devices supported by the LVFS.

Works for me, thanks!

Comment (mendel5) [reply lvfs/devices/templates/devicelist.html]: Ok, I updated my pull request.

**Comment by mendel5** [comment]
Thanks!
